package.sh

```diff
@@ -157,16 +163,25 @@ else
 	##    - building binaries from sources
 	##    - copy artifacts to the build_dir
 	echo "Build the module..."
-	if [ -e ./packaging/build.sh ]; then
-		./packaging/build.sh $PWD ${build_dir} || exit 1
-	else
+	if [ ! -e ./packaging/build.sh ]; then
 		echo "ERROR: No build script available"
 		exit 1
 	fi
+	if [ "${kernel_config}" == "" ]; then
+		./packaging/build.sh $PWD ${build_dir} || exit 1
+	else
+		### Kernel build needs to know which configuration to be built.
+		./packaging/build.sh $PWD ${kernel_config} || exit 1
```

Comment:
Could this be done with one call, so that the config param would be the third param? If it is not used it can be just empty string.
./packaging/build.sh $PWD ${packaging_dir} ${config}
(this example assumes "build_dir" param is renamed to more describing "packaging_dir")

Reply:
Also, if/when kernel build script does not need any separate temporary directory from where it generates the debian package, it can just ignore the "build_dir" (or the deb_dir if renamed) param.

Reply:
In Slack chat it has been agreed to use tmp_deb_dir instead build_dir. The tmp_deb_dir points to the directory containing the files that will be packed in a debian package.
The kernel build script ignores this directory because it does not use debian temporary directory for creating the debian package. This is ok.